src/components/tl/mlx5/mcast/tl_mlx5_mcast_team.c

```diff
@@ -9,14 +9,184 @@
 #include "tl_mlx5_mcast_coll.h"
 #include "coll_score/ucc_coll_score.h"
 #include "tl_mlx5_mcast_helper.h"
+#include "p2p/ucc_tl_mlx5_mcast_p2p.h"
+#include "mcast/tl_mlx5_mcast_helper.h"
+ 
+static ucc_status_t ucc_tl_mlx5_mcast_service_bcast_post(void *arg, void *buf, int size, int root,
+                                                         ucc_service_coll_req_t **bcast_req)
+{
+    ucc_tl_mlx5_mcast_oob_p2p_context_t *ctx    = (ucc_tl_mlx5_mcast_oob_p2p_context_t *)arg;
+    ucc_status_t                         status = UCC_OK;
+    ucc_team_t                          *team   = ctx->base_team;
```

Comment:
Why use full team from context and not the current team ?

Reply:
all these team info are taken from ````params```` input of ```ucc_tl_mlx5_team_t``` team create funtion:
````
UCC_CLASS_INIT_FUNC(ucc_tl_mlx5_team_t, ucc_base_context_t *tl_context,
                    const ucc_base_team_params_t *params)
````
I believe this ```params``` input is reflecting the subcomm, not comm world. Please take a look at the ```ucc_tl_mlx5_mcast_team_init``` for further info.